templates/CRM/Profile/Page/MultipleRecordFieldsListing.tpl

```diff
@@ -98,7 +98,7 @@
     <div id='{$dialogId}' class="hiddenElement"></div>
   {/if}
 
-  {if !$reachedMax}
+  {if !$reachedMax && $showAddButton}
```

Comment:
We have been making an effort to eliminate references to undefined variables in our `.tpl` files, and I note that this variable is not always set. It could be fixed by changing this line to:
```suggestion
  {if empty($reachedMax) && !empty($showAddButton)}
```